src/Place/CanonicalPlaceRepository.php

```diff
@@ -4,24 +4,20 @@
 
 namespace CultuurNet\UDB3\Place;
 
-class CanonicalPlaceRepository
+use CultuurNet\UDB3\Place\Canonical\DuplicatePlaceRepository;
+
+final class CanonicalPlaceRepository
```

Comment:
I wonder if we should not remove this entire class and just add **DuplicatePlaceRepository** as a dependency to **RelocateEventToCanonicalPlace**.
This class is now just a single line wrapper for **DuplicatePlaceRepository**, and only used in 1 place.

Reply:
Indeed, will remove it.